src/Kobo/SystemDialog.cpp

```diff
@@ -40,6 +40,8 @@ class SystemWidget final
     USB_STORAGE,
   };
 
+  Button *switch_kernel_button;
```

Comment:
This variable is only used inside `Prepare()`, so better make it a local variable.